### ISSUE TITLE
Install TLS certificate for keepachangelog.com

### ISSUE THREAD

**Issue by mpbzh** [olivierlacan/keep-a-changelog#96]
Suggestion: Install a TLS certificate for keepachangelog.com so it can also be accessed via https://keepachangelog.com

**Comment by olivierlacan** [comment]
Good idea. I've been meaning to look at [Let's Encrypt](https://letsencrypt.org/).

**Comment by mpbzh** [comment]
I was going to suggest Let's Encrypt, I'm successfully using certs issued by them on a number of domains.

**Comment by nbibler** [comment]
Unless I'm mistaken, GitHub Pages doesn't allow for custom SSL certificates. In order to HTTPS the domain, it would have to find new hosting (not difficult, just another task to perform).

It may also be possible to use CloudFlare (or similar) with a custom SSL certificate in front of GH Pages…

**Comment by olivierlacan** [comment]
I don't want to use CloudFlare for any of my projects since they block all Tor traffic. I'd much rather set up something semi-static on Heroku.

**Comment by TheLastProject** [comment]
For what it's worth, you can [explicitly unblock Tor in CloudFlare by whitelisting the T1 area](https://support.cloudflare.com/hc/en-us/articles/203306930-Does-CloudFlare-block-Tor). But the defaults for Tor with CloudFlare are indeed problematic. I have no preference for CloudFlare, Heroku is probably a nicer choice, but I just figured I'd chime in with this bit of info.

**Comment by Ajedi32** [comment]
That page actually says they _don't_ block Tor traffic by default. If traffic from a particular exit node has been misbehaving, then users accessing the site through that node may have to complete a CAPTCHA before they can proceed, but they're not outright blocked.

**Comment by ryansully** [comment]
If you use CloudFront you can get a certificate for your distribution that covers the apex/root domain (keepachangelog.com) as well as wildcard for any subdomain (*.keepachangelog.com). The cert is free and automatically renews, but can only be used for CloudFront distributions and Elastic Load Balancing.

**Comment by olivierlacan** [comment]
Can't remember when this change occurred but keepachangelog.com has used TLS for a while now. Using Let's Encrypt, not Cloudflare. 
![Uploading image.png…]()
